packages/twenty-shared/src/workflow/utils/getWorkflowRunContext.ts

```diff
@@ -1,12 +1,12 @@
-import { isDefined } from '@/utils';
```

Comment:
I don't feel like this kind of changes should be done in a workflow PR. If you think we have to do them, can you please do them in a separate PR where you explicit the motive?